src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckClassVisitor.kt

```diff
@@ -192,4 +193,6 @@ private class WrapMethodInIgnoredSectionTransformer(
         }
         visitInsn(opcode)
     }
-}
\ No newline at end of file
+}
+
+internal val coroutineCallingClasses = HashSet<String>()
```

Comment:
What is this `String`?

Reply:
I added a comment explaining this variable.

Reply:
I don't see the comment in the code. Could you check it on your side, please?

Reply:
Can you please check if you see this commit?
https://github.com/JetBrains/lincheck/pull/316/commits/ea912a746c3bc8ea9c72008e984922d0ed85dcb5

Reply:
Yes, now I see that